### PR TITLE
[11.0][MIG] account_voucher_killer

### PR DESCRIPTION
**Pull request by astirpe**
Moved from https://github.com/OCA/bank-payment/pull/604

### CODE REVIEW

File: account_voucher_killer/view/invoice_view.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="invoice_payment_group_customer_invoice" model="ir.ui.view">
```
Comment (tbaden) [inline account_voucher_killer/view/invoice_view.xml:4]: optional change:
use original id (invoice_form)

> prefer naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name.

File: account_voucher_killer/view/invoice_view.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+           <attribute name="groups">account_voucher_killer.invoice_payment_user</attribute>
+        </xpath>
+        <xpath expr="//button[@name='%(account.action_account_invoice_payment)d']"
+               position="attributes">
+           <attribute name="groups">account_voucher_killer.invoice_payment_user</attribute>
+        </xpath>
+      </field>
+    </record>
+
+    <record id="invoice_payment_group_supplier_invoice" model="ir.ui.view">
```
Comment (tbaden) [inline account_voucher_killer/view/invoice_view.xml:20]: optional change:
use original id (invoice_supplier_form)

> prefer naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name.

File: account_voucher_killer/README.rst [hunk 3]
```diff
@@ -1,8 +1,29 @@
...
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-
-Prevent the usage of payments from invoices
-===========================================
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fbank--payment-lightgray.png?logo=github
+    :target: https://github.com/OCA/bank-payment/tree/11.0/account_voucher_killer
+    :alt: OCA/bank-payment
```
Comment (luismontalba) [inline account_voucher_killer/README.rst]: The repo referred in the PR in several places is still bank-payment instead of account-payment

Comment (astirpe) [reply account_voucher_killer/README.rst]: @luismontalba readme updated. Thanks!

File: account_voucher_killer/__manifest__.py [hunk 4]
```diff
@@ -1,28 +1,12 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 {'name': 'Accounting Payment Access',
- 'version': '10.0.1.0.0',
+ 'summary': 'Prevent the usage of payments from invoices',
+ 'version': '11.0.1.0.0',
  'category': 'other',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
+ 'author': 'Camptocamp,Odoo Community Association (OCA)',
  'website': 'https://github.com/OCA/bank-payment',
```
Comment (luismontalba) [inline account_voucher_killer/__manifest__.py]: bank-payment -> account-payment

File: account_voucher_killer/readme/INSTALL.rst [hunk 5]
```diff
@@ -0,0 +1,5 @@
+This module depends on :
+
+* account
+
+This modules are parts of the OCA/bank-payment suite.
```
Comment (luismontalba) [inline account_voucher_killer/readme/INSTALL.rst]: bank-payment -> account-payment

**Review by luismontalba — CHANGES_REQUESTED** [review]

**Comment by luismontalba** [comment]
Shouldn't be these menus hidden too?

DeepinScreenshot_Seleccionar área_20190524141301

**Comment by astirpe** [comment]
@luismontalba not sure if those menu items should be hidden, anyway they are provided by other modules like `account_payment_return`: they are not menu items of the standard Odoo.

File: account_voucher_killer/readme/INSTALL.rst (hunk 5 above)
Comment (astirpe) [reply account_voucher_killer/readme/INSTALL.rst]: :+1: thanks!

File: account_voucher_killer/__manifest__.py (hunk 4 above)
Comment (astirpe) [reply account_voucher_killer/__manifest__.py]: :+1:

**Comment by luismontalba** [comment]
OK, I see now

**Review by luismontalba — APPROVED** [review]

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM
some optional changes if you like

**Pull request merged**